pkg/test/resourcefixture/testdata/basic/containerattached/v1beta1/containerattachedclusterbasic/_http.log

```diff
@@ -375,13 +224,18 @@ Content-Type: application/json
 User-Agent: Terraform/ (+https://www.terraform.io) Terraform-Plugin-SDK/2.10.1 terraform-provider-google-beta/kcc/controller-manager
 
 {
+  "binaryAuthorization": {
+    "evaluationMode": "DISABLED"
+  },
```

Comment:
ditto: Could you update the mockgcp to match the real GCP for this field? We want to guardrail the default values given from the GCP service (otherwise KCC could overrule this)